### ISSUE TITLE
Gradle 7 error on publishing

### ISSUE THREAD

**Issue by robvadai** [ADTRAN/gradle-scala-multiversion-plugin#35]
When using Gradle 7 and the Maven Publish plugin, I get:

```
Execution failed for task ':myproject:generatePomFileForApiPublication'.
> Could not apply withXml() to generated POM
   > Could not get unknown property 'compile' for configuration container of type org.gradle.api.internal.artifacts.configurations.DefaultConfigurationContainer.
```

Removing this plugin solves the issue. So I think the plugin should be using a different property here: https://github.com/ADTRAN/gradle-scala-multiversion-plugin/blob/2c7db90802fe677c3e6aca03a2c3d7b963fdff17/src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy#L171

To replicate, run:

```sh
./gradlew clean generatePomFileForApiPublication -p myproject
```

I can confirm it's an issue in versions `1.0.37` and `1.0.38-main-SNAPSHOT`.

**Comment by nathanalderson** [comment]
Thanks for reporting. I actually have a fix for this sitting in a local WIP branch. It's tied up in migrating the project to Github Actions and publishing to OSSRH instead of JFrog Bintray. I almost have https://github.com/ADTRAN/gradle-betablocker-plugin migrated but I'm waiting on an internal team to get my publish credentials fixed. Anyway, once that one is finished, I should get this one done fairly quickly.

**Comment by nathanalderson** [comment]
This should be fixed once #36 is merged and any resulting fallout is fixed.

**Comment by nathanalderson** [comment]
This should be fixed in version 2.0.4 or later. Please re-open this issue if you still have trouble.

**Comment by robvadai** [comment]
New version works, all good. Thanks for the fix!
